manifests/latest/vsphere-67u3/vanilla/rbac/vsphere-csi-controller-rbac.yaml

```diff
@@ -27,6 +27,12 @@ rules:
   - apiGroups: ["storage.k8s.io"]
     resources: ["volumeattachments"]
     verbs: ["get", "list", "watch", "update", "patch"]
+  - apiGroups: ["cns.vmware.com"]
```

Comment:
Is migration supported in 67u3? I think this only needs to be added to 70u1 rules

Reply:
+1

Reply:
for now, let's just update 7.0u1. if migration-related changes will be backported to vCenter 67. we will update it later.